### PR TITLE
Basic UI Changes Update

### PR DESCRIPTION
**Pull request by LefanTan**
Made changes to string.xml to add a basic string array. Made changes to build gradle file too

### CODE REVIEW

File: app/src/main/res/values/font_certs.xml [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
...cm9pZDEQMA4GA1UECxMHQW5kcm9pZDEQMA4GA1UEAxMHQW5kcm9pZDEiMCAGCSqGSIb3DQEJARYTYW5kcm9pZEBhbmRyb2lkLmNvbTCCASAwDQYJKoZIhvcNAQEBBQADggENADCCAQgCggEBANbOLggKv+IxTdGNs8/TGFy0PTP6DHThvbbR24kT9ixcOd9W+EaBPWW+wPPKQmsHxajtWjmQwWfna8mZuSeJS48LIgAZlKkpFeVyxW0qMBujb8X8ETrWy550NaFtI6t9+u7hZeTfHwqNvacKhp1RbE6dBRGWynwMVX8XW8N1+UjFaq6GCJukT4qmpN2afb8sCjUigq0GuMwYXrFVee74bQgLHWGJwPmvmLHC69EH6kWr22ijx4OKXlSIx2xT1AsSHee70w5iDBiK4aph27yH3TxkXy9V89TDdexAcKk/cVHYNnDBapcavl7y0RiQ4biu8ymM8Ga/nmzhRKya6G0cGw8CAQOjgfwwgfkwHQYDVR0OBBYEFI0cxb6VTEM8YYY6FbBMvAPyT+CyMIHJBgNVHSMEgcEwgb6AFI0cxb6VTEM8YYY6FbBMvAPyT+CyoYGapIGXMIGUMQswCQYDVQQGEwJVUzETMBEGA1UECBMKQ2FsaWZvcm5pYTEWMBQGA1UEBxMNTW91bnRhaW4gVmlldzEQMA4GA1UEChMHQW5kcm9pZDEQMA4GA1UECxMHQW5kcm9pZDEQMA4GA1UEAxMHQW5kcm9pZDEiMCAGCSqGSIb3DQEJARYTYW5kcm9pZEBhbmRyb2lkLmNvbYIJANWFuGx90071MAwGA1UdEwQFMAMBAf8wDQYJKoZIhvcNAQEEBQADggEBABnTDPEF+3iSP0wNfdIjIz1AlnrPzgAIHVvXxunW7SBrDhEglQZBbKJEk5kT0mtKoOD1JMrSu1xuTKEBahWRbqHsXclaXjoBADb0kkjVEJu/Lh5hgYZnOjvlba8Ld7HCKePCVePoTJBdI4fvugnL8TsgK05aIskyY0hKI9L8KfqfGTl1lzOv2KoWD0KWwtAWPoGChZxmQ+nBli+gwYMzM1vAkP+aayLe0a1EQimlOalO762r0GXO0ks+UeXde2Z4e+8S/pf7pITEI/tP+MxJTALw9QUWEv9lKTk+jkbqxbsh8nfBUapfKqYn0eidpwq2AzVp3juYl7//fKnaPhJD9gs=
```
Comment (liushuyu) [inline app/src/main/res/values/font_certs.xml:9]: Is this some tokens from your account or something?

Comment (LefanTan) [reply app/src/main/res/values/font_certs.xml]: This was generated when we added font through android studio

Comment (LefanTan) [reply app/src/main/res/values/font_certs.xml:9]: This was added after we added fonts through android studio, I think they generated this for us.

File: app/src/main/java/io/github/cmput301f19t19/legendary_fiesta/ui/OwnMoodsFragment.java [hunk 2]
```diff
@@ -12,15 +18,43 @@
...
 
         View root = inflater.inflate(R.layout.fragment_own_moods, container, false);
+        mView = root;
 
+        setUpSpinner();
         // Information from FireBase
-        TextView textView = root.findViewById(R.id.text_my_moods);
-        textView.setText(requireActivity().getIntent().getStringExtra("USER_NAME"));
-
+        //TextView textView = root.findViewById(R.id.text_my_moods);
```
Comment (liushuyu) [inline app/src/main/java/io/github/cmput301f19t19/legendary_fiesta/ui/OwnMoodsFragment.java]: Why is this commented out?

File: .idea/misc.xml [hunk 3]
```diff
@@ -1,6 +1,7 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <project version="4">
   <component name="ProjectRootManager" version="2" languageLevel="JDK_1_7" default="false" project-jdk-name="JDK" project-jdk-type="JavaSDK">
+
```
Comment (liushuyu) [inline .idea/misc.xml]: Unintended changes?

**Review by liushuyu — CHANGES_REQUESTED** [review]
Comments left.

Your PR description is too vague for a change like this. Be more descriptive and meaningful; although this is a school project but try to make it a good habit.

If you don't know what to write, try to ask yourself questions like:

- Why is this change needed?
- What have I done to implement this change?
- (If the change is very drastic or complicated) How I did it and why I chose this method?
- What are the potential issues?

The PR itself seems good on the first glance. I will take a good look later.

File: app/src/main/java/io/github/cmput301f19t19/legendary_fiesta/ui/OwnMoodsFragment.java (hunk 2 above)
Comment (LefanTan) [reply app/src/main/java/io/github/cmput301f19t19/legendary_fiesta/ui/OwnMoodsFragment.java]: this was your code, i will remove it.

File: .idea/misc.xml (hunk 3 above)
Comment (LefanTan) [reply .idea/misc.xml]: fixed

**Review by liushuyu — APPROVED** [review]
LGTM

**Pull request merged**